tests/unit/python/foglamp/services/south/test_ingest.py

```diff
@@ -0,0 +1,588 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test services/south/ingest.py
+
+"""
+import copy
+import pytest
+from unittest.mock import MagicMock
+from foglamp.services.south.ingest import *
+from foglamp.services.south import ingest
```

Comment:
re-think about these 2 imports

Reply:
required

Reply:
that should be available as `__LOGGER` with import * (no?)

Reply:
Unfortunately, not. It is not available with import *, yet to find a reason for it.